### PR TITLE
Add suppport for `ps` type

### PR DESCRIPTION
**Pull request by arthurvr**
postscript detection :cake:

https://github.com/sindresorhus/file-type/pull/18#issuecomment-77688922

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +58,7 @@
     "rar",
     "gz",
     "bz2",
+    "ps",
```
Comment (sindresorhus) [inline package.json]: I know this is crazy, but can you put it at the end so everything is in the same order?

File: index.js [hunk 2]
```diff
@@ -4,6 +4,13 @@ module.exports = function (buf) {
 		return null;
 	}
 
+	if (buf[0] === 0x25 && buf[1] === 0x21) {
+		return {
+			ext: 'ps',
+			mime: 'application/postscript'
+		};
+	}
+
```
Comment (sindresorhus) [inline index.js]: Can you put this test at the end too?

Comment (arthurvr) [inline index.js]: done

File: package.json (hunk 1 above)
Comment (arthurvr) [inline package.json]: done

**Pull request merged**